packages/react/src/components/Card/index.tsx

```diff
@@ -64,7 +64,7 @@ export const Card = React.forwardRef(({ children, flowId, ...props }: CardProps,
 Card.Dismiss = (props: ButtonProps) => {
   return (
     <Button.Plain part="dismiss" padding={0} {...props}>
-      <XMarkIcon height="24" fill="currentColor" />
+      <XMarkIcon height="20" fill="currentColor" />
```

Comment:
If you swap the import for `XMarkIcon` to the 20x20 version of HeroIcons this will keep the correct scale / stroke.

Reply:
Niice